src/clr-angular/clr-angular.module.ts

```diff
@@ -19,6 +19,7 @@ import { ClrIfExpandModule } from './utils/expand/if-expand.module';
 import { ClrFocusTrapModule } from './utils/focus-trap/focus-trap.module';
 import { ClrLoadingModule } from './utils/loading/loading.module';
 import { ClrWizardModule } from './wizard/wizard.module';
+import { ClrSmartPopoverModule } from './utils/smart-popover/smart-popover.module';
```

Comment:
I'm not sure, SmartPopover implements conditional behavior but as I work on the positioning it seems like by the end it is going to have way more that just a directive and a service. The conditional entities only have a single directive and service. I'm going to leave it here for now b/c this can be revisited as the feature gets more solid.